README.md

```diff
@@ -125,14 +147,43 @@ github_googlesheet:
 
 ### Setting up Google Analytics
 
+[Follow the steps from the above section to authenticate Google](#setting-up-google-authentication) -- make sure that the account you have authenticated for has access to the Google Analytics properties you wish to collect.
```

Comment:
What's the "above section" referring to here?

Reply:
I see it in the TOC, but I don't remember seeing/reading the section itself

Reply:
This should be a link  to `#setting-up-google-authentication` which is: https://github.com/fhdsl/metricminer-dashboard?tab=readme-ov-file#setting-up-google-authentication

Reply:
Oh - perfect -- thanks! 
In that case, do you want to edit the first sentence in that section "Your metricminer dashboard will need permissions to save to your Google drive." so that it's clear that setting up the google authentication isn't just for saving results to google drive (since saving to google drive is optional)?